charts/jenkins/README.md

```diff
@@ -517,6 +517,16 @@ master:
       enabled: true       # was false
 ```
 
+## Prometheus Metrics
```

Comment:
Can you move this before `### HTTPS Keystore Configuration`.
Here it would be part of the migration to version 2.0.0.